user_profile/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+# Create your models here.
+
+
+class UserProfile(models.Model):
+    user = models.OneToOneField(User, related_name="profile", on_delete=models.CASCADE)
+    cinc_number = models.CharField(max_length=20, blank=True, verbose_name="CNIC Number")
+    occupation = models.CharField(max_length=500, blank=True, verbose_name="Your Occupation")
+    bio = models.TextField(max_length=500, blank=True, verbose_name="About (Bio)")
+    address = models.CharField(max_length=255, blank=True, verbose_name="Permanent Address")
+    contact_number = models.CharField(max_length=30, blank=True, verbose_name="Contact Number")
+    city = models.CharField(max_length=30, blank=True, verbose_name="Your Location (City)")
+    country = models.CharField(max_length=30, blank=True, verbose_name="Your Nationality (Country)")
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    @receiver(post_save, sender=User)
```

Comment:
both of these signal receivers don't make sense to me. We probably use `post_save` signal to send user welcome email.

Reply:
1st signal receiver (**create_user_profile**): if the newly user-created via signup it receives user profile fields and creates a user profile for the newly created user. I'll add sending welcome functionally this signal. 

2nd signal receiver (**save_user_profile**): if the user updated then the user profile update its updated_at field via user profile instance.

Reply:
For now, I've removed both receivers. I'll add a post_save signal for sending a welcome email to newly created User in the next PR.